jenkins-master/jobs/scripts/workspace/requirements.txt

```diff
@@ -6,8 +6,14 @@ treeherder-client==2.0.1
 # Required by mozinfo
 mozfile==1.2
 
+# https://github.com/mozilla/mozmill-ci/issues/628 (and 831)
+# As long as Python 2.7.10 cannot be used on our master machines
+ndg-httpsclient
+pyasn1
+pyopenssl
```

Comment:
Dave, the PR has been updated for pinned versions which are proven to work on the CI master machines for a while.